urwidtrees/tree.py

```diff
@@ -137,6 +137,18 @@ def Posgen(reverse):
                     current = self.next_position(current)
         return Posgen(reverse)
 
+    def leaves(self, pos):
+        """returns a recursive list of leaves below `pos`"""
```

Comment:
I am not sure if I understand the code correctly but to me it looks like the list is not recursive. Only the calculation of the list is.